_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -18,77 +18,133 @@ acknowledgements:
 
-**Note**: Leading and trailing [whitespace](#whitespace) and difference in case sensitivity should be ignored.
+**Note:** Due to the definition of [included characters](#included-characters) the whole consecutive sequence of characters in the [visible](#visible) [descendant text content](https://dom.spec.whatwg.org/#concept-descendant-text-content) of the target element has to be included character-by-character (excluding [punctuation](#punctuation)) in the [accessible name](#accessible-name) to meet the expectation of this rule. The [accessible name](#accessible-name) is however allowed to also contain text content other than the [visible](#visible) [descendant text content](https://dom.spec.whatwg.org/#concept-descendant-text-content).
 
 ## Assumptions
 
-_There are currently no assumptions_
+- This rule assumes that the [visible](#visible) [text nodes](https://www.w3.org/TR/dom/#text) follow the same order as the text in the code, e.g. CSS has not been used to re-arrange pieces of text so that they are presented in a different order.
+- This rule assumes that the whole set of [visible](#visible) [text nodes](https://www.w3.org/TR/dom/#text) of the element are styled, including placement, in such a way that a sighted user will perceive them all as part of the label of the element. An example where this would not be the case, is where styling is used to split up the visible text nodes in e.g. two parts and place one part further away, or give it a different styling that makes it appear as not connected to the element it labels.
 
 ## Accessibility Support
 
-There are no major accessibility support issues known for this rule.
+_No accessibility support issues known._
 
 ## Background
 
 - [Understanding Success Criterion 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)
+- [Technique G208: Including the text of the visible label as part of the accessible name](https://www.w3.org/WAI/WCAG21/Techniques/general/G208)
+- [Technique G211: Matching the accessible name to the visible label](https://www.w3.org/WAI/WCAG21/Techniques/general/G211)
 
 ## Test Cases
 
 ### Passed
 
 #### Passed Example 1
 
-[Visible][] label and [accessible name][] matches when trailing white spaces are removed.
+[Visible][] [label][] and [accessible name][] match when trailing white spaces are removed.
 
 ```html
 <div role="link" aria-label="next page ">next page</div>
 ```
 
 #### Passed Example 2
 
-Character insensitivity between [visible][] label and [accessible name][].
+[Visible][] [label][] and [accessible name][] match when ignoring case sensitivity.
```

Comment:
match => link to def (to make obvious the fact that case sensitivity is ignored when matching).